data/human/jobs.txt

```diff
@@ -3781,3 +3781,110 @@ mission "Hunted"
 		system
 			distance 5 5
 		fleet "Marauder fleet X"
+
+
+mission "Republic Navy Advisory System [frozen]"
+	name "Cold Relief"
+	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe cold snap."
+	minor
+	repeat
+	deadline
+	source
+		government "Republic"
+	destination
+		government "Republic" "Syndicate" "Independent" "Neutral"
+		attributes "frozen"
+	substitutions
+		"<problem>" "cold snap"
+		"<problem>" "freeze"
+			"random" < 60
+		"<problem>" "snowstorm"
+			"random" < 30
+		"<problem>" "blizzard"
+			"random" < 10
+	cargo "cold aid" 10 15 0.5
+	to offer
+		random < 1
+	on offer
+		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
+	on complete
+		payment 8000
+		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`
+
+mission "Republic Navy Advisory System [volcanic]"
+	name "Volcanic Relief"
+	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe volcanic eruption."
+	minor
+	repeat
+	deadline
+	source
+		government "Republic"
+	destination
+		government "Republic" "Syndicate" "Independent" "Neutral"
```

Comment:
The way you have these missions set up, you can get unnecessarily long trips to your destination. One possible mission might have a player go from Wayfarer to Calda and and get only 8000 credits for it. You should consider adding a distance filter up to 8 systems away.
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```